### PR TITLE
Add login

### PR DESCRIPTION
**Pull request by jeanpimentel**

### CODE REVIEW

File: app/src/main/java/com/ciandt/dragonfly/example/features/login/LoginContract.kt [hunk 1]
```diff
@@ -0,0 +1,41 @@
+package com.ciandt.dragonfly.example.features.login
+
+import com.ciandt.dragonfly.example.shared.BasePresenterContract
+import com.google.android.gms.auth.api.signin.GoogleSignInResult
+import com.google.firebase.auth.FirebaseUser
+
+/**
+ * Created by iluz on 6/20/17.
```
Comment (ivamluz) [inline app/src/main/java/com/ciandt/dragonfly/example/features/login/LoginContract.kt]: :)

File: app/src/main/java/com/ciandt/dragonfly/example/features/login/LoginPresenter.kt [hunk 2]
```diff
@@ -0,0 +1,47 @@
+package com.ciandt.dragonfly.example.features.login
+
+import com.ciandt.dragonfly.example.shared.BasePresenter
+import com.google.android.gms.auth.api.signin.GoogleSignInResult
+
+/**
+ * Created by iluz on 6/20/17.
```
Comment (ivamluz) [inline app/src/main/java/com/ciandt/dragonfly/example/features/login/LoginPresenter.kt]: :)

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -59,4 +59,12 @@
...
     <string name="feedback_save_image_success">Image saved successfully.</string>
     <string name="feedback_save_image_error">Something was wrong. Please try again.</string>
+
+    <string name="login_with_google">LOGIN WITH GOOGLE</string>
+    <string name="login_title">Dragonfly Lens</string>
+    <string name="login_subtitle">Send feedback and compare the results with other services.
+    </string>
+    <string name="login_terms_and_conditions">TERMS AND CONDITIONS</string>
+
+    <string name="login_error">There was an error when trying to connect. Please, try again.</string>
```
Comment (ivamluz) [inline app/src/main/res/values/strings.xml]: I think we can remove this "," after "Please". In pt-br, that would be mandatory. In English, I think it more forceful: https://english.stackexchange.com/a/391

**Review by ivamluz — COMMENTED** [review]
- There are some comments that seem to have been copied and pasted.
- There is a string that I think we can change.

**Comment by jeanpimentel** [comment]
Reviewed

**Pull request merged**